include/pacmod_game_control/globals.h

```diff
@@ -103,7 +103,7 @@ static const float ACCEL_SCALE_FACTOR = 0.6;
 static const float ACCEL_OFFSET = 0.21;
 static const float STEER_SCALE_FACTOR = 1.5;
 static const float STEER_OFFSET = 1.0;
-static const float MAX_ROT_RAD_VEHICLE2 = 6.5;
+static const float MAX_ROT_RAD_VEHICLE2 = 8.5;
```

Comment:
Is this the actual maximum rotation range for this vehicle? If not, this should instead be set in startup_checks.cpp in the controller selection section. When the vehicle is selected, it sets the value PublishControl::max_rot_rad. This value should be the one modified when the controller type is selected.

Reply:
Yes its the actual maximum value, im not sure why it was set at this for vehicle.